pkg/systemd/terminal.jsx

```diff
@@ -94,12 +105,12 @@ const _ = cockpit.gettext;
                     <div className="panel-heading terminal-group">
                         <tt className="terminal-title">{this.state.title}</tt>
                         <div>
-                            <Select onChange={ this.onThemeChanged }
-                                    id="theme-select"
-                                    initial={ this.state.theme }>
-                                <SelectEntry data='dark-theme'>{ _("Dark") }</SelectEntry>
-                                <SelectEntry data='light-theme'>{ _("Light") }</SelectEntry>
-                            </Select>
+                            <StatelessSelect id="theme-select" iconClass="fa fa-cog">
+                                <span className="terminal-group">
+                                    <button className={ 'theme-btn dark-theme ' + this.state.darkTheme } onClick={() => this.onThemeChanged('dark-theme') } />
+                                    <button className={ 'theme-btn light-theme ' + this.state.lightTheme } onClick={() => this.onThemeChanged('light-theme') } />
```

Comment:
While that looks kind of cool, I think we should have a standard vertical selection with theme names. Not everyone might be able to see the fine nuances in the colors in the small selector.

Reply:
I will wait for review from @garrett . In the case he agrees with having simple dropdown selection, then I will drop last commits (and add two themes). That also will resolve almost all of your finds, so until then I will not resolve those.